Loop/Managers/WatchDataManager.swift

```diff
@@ -369,7 +369,7 @@ final class WatchDataManager: NSObject {
                 return
             }
 
-            deviceManager.enactBolus(units: bolus.value, automatic: false) { (error) in
+            deviceManager.enactBolus(units: bolus.value, activationType: .manualNoRecommendation) { (error) in
```

Comment:
@ps2 I'm concerned that `.manualNoRecommendation` isn't correct and I'm overlooking the bolus recommendation.

Reply:
To be more clear, the dosing decision is sent from the watch, and you can use this to decide if it should be manual accepted or manual adjusted, or no recommendation.